packages/hoppscotch-common/src/components/collections/index.vue

```diff
@@ -860,6 +861,7 @@ const editRequest = (payload: {
 }) => {
   const { folderPath, requestIndex, request } = payload
   editingRequest.value = request
+  editingRequestName.value = request.name || ""
```

Comment:
Why the OR assignment ?

`request.name` cannot be a nullish value (it is typed as a string).